config/initializers/_load_config.rb

```diff
@@ -96,6 +91,6 @@
   default.reverse_merge!(options_from_config)
 end
 
-if Rails.env.production?
-  Rails.application.config.consider_all_requests_local = Errbit::Config.display_internal_errors
-end
+Rails.application.config.consider_all_requests_local = Errbit::Config.display_internal_errors
```

Comment:
After playing with this idea, I've convinced myself that these named environment files are not meaningless. I thought I would find a lot more similarities between between named environments.

Here's what I'm thinking now. We should allow people to configure via environment variables anything that might change between deploys within the same named environment. For example this would include serve_static_assets, log_level, log_location and secret_key_base.

I know rails likes to do the erb -> yaml dance for secret_key_base in a special file called secrets. I don't understand why that's a good idea.